README.md

```diff
@@ -7,8 +7,56 @@ https://www.youtube.com/watch?v=GKPdwjFkoxY
 Description coming soon!
 
 ### Endpoints
-1. `/all-states` - returns a list of all the latest states currently held by the aggregate node  
-Example: https://contracts.warp.cc/all-states?orderBy=sort_key&order=desc&limit=5
-2. `/balances` - returns a list of all balances held by a given `walletAddress`  
-Example: https://contracts.warp.cc/balances?walletAddress=C6nBlkd7pKRxPiUk1AaGeCrup7XiVQow40NO6c9RDGg
 
+1. `/all-states` - returns a list of all the latest states currently held by the
+   aggregate node Example:
+   https://contracts.warp.cc/all-states?orderBy=sort_key&order=desc&limit=5
+2. `/balances` - returns a list of all balances held by a given `walletAddress`
+   Example:
+   https://contracts.warp.cc/balances?walletAddress=C6nBlkd7pKRxPiUk1AaGeCrup7XiVQow40NO6c9RDGg
+3. `/interactions-by-indexes` - returns a list off interactions associated with
```

Comment:
btw., what is added value of this solution over simply using the arweave.net GQL endpoint (and searching by tag values)? response times? risk of being rate limitted?

Reply:
Disscused on daily + https://github.com/warp-contracts/warp-aggregate-node/issues/1#issuecomment-1408532659